--- libsmi-0.5.0/mibs/iana/Makefile.am~	2014-08-15 07:32:46.000000000 +0000
+++ libsmi-0.5.0/mibs/iana/Makefile.am	2022-09-27 15:00:00.851667841 +0000
@@ -11,7 +11,7 @@
 # @(#) $Id: Makefile.am 1810 2014-08-15 07:32:45Z schoenw $
 #
 
-ianadir			= $(mibdir)/iana
+ianadir			= $(mibdir)
 MIBS			= $(shell ls -1 $(srcdir) | egrep -v 'CVS|Makefile')
 EXTRA_DIST		= $(MIBS)
 iana_DATA		= $(MIBS)
--- libsmi-0.5.0/mibs/ietf/Makefile.am~	2013-06-04 12:33:59.000000000 +0000
+++ libsmi-0.5.0/mibs/ietf/Makefile.am	2022-09-27 15:01:39.373255567 +0000
@@ -11,7 +11,7 @@
 # @(#) $Id: Makefile.am 1104 2003-12-01 12:24:15Z strauss $
 #
 
-ietfdir			= $(mibdir)/ietf
+ietfdir			= $(mibdir)
 MIBS			= $(shell ls -1 $(srcdir) | egrep -v 'CVS|Makefile')
 EXTRA_DIST              = $(MIBS)
 ietf_DATA		= $(MIBS)
--- libsmi-0.5.0/mibs/site/Makefile.am~	2013-06-04 12:33:59.000000000 +0000
+++ libsmi-0.5.0/mibs/site/Makefile.am	2022-09-27 15:02:21.233978119 +0000
@@ -11,7 +11,7 @@
 # @(#) $Id: Makefile.am 1105 2003-12-01 19:01:38Z strauss $
 #
 
-sitedir			= $(mibdir)/site
+sitedir			= $(mibdir)
 MIBS			= $(shell ls -1 $(srcdir) | egrep -v 'CVS|Makefile')
 EXTRA_DIST              = $(MIBS)
 site_DATA		= $(MIBS)
--- libsmi-0.5.0/mibs/tubs/Makefile.am~	2013-06-04 12:33:59.000000000 +0000
+++ libsmi-0.5.0/mibs/tubs/Makefile.am	2022-09-27 15:02:57.731295538 +0000
@@ -11,7 +11,7 @@
 # @(#) $Id: Makefile.am 1105 2003-12-01 19:01:38Z strauss $
 #
 
-tubsdir			= $(mibdir)/tubs
+tubsdir			= $(mibdir)
 MIBS			= $(shell ls -1 $(srcdir) | egrep -v 'CVS|Makefile')
 EXTRA_DIST              = $(MIBS)
 tubs_DATA		= $(MIBS)
--- libsmi-0.5.0/mibs/irtf/Makefile.am~	2013-06-04 12:33:59.000000000 +0000
+++ libsmi-0.5.0/mibs/irtf/Makefile.am	2022-09-27 15:04:39.206528730 +0000
@@ -11,7 +11,7 @@
 # @(#) $Id: Makefile.am 1105 2003-12-01 19:01:38Z strauss $
 #
 
-irtfdir			= $(mibdir)/irtf
+irtfdir			= $(mibdir)
 MIBS			= $(shell ls -1 $(srcdir) | egrep -v 'CVS|Makefile')
 EXTRA_DIST              = $(MIBS)
 irtf_DATA		= $(MIBS)
--- libsmi-0.5.0/pibs/Makefile.am~	2013-06-04 12:34:00.000000000 +0000
+++ libsmi-0.5.0/pibs/Makefile.am	2022-09-27 15:05:40.017723362 +0000
@@ -16,8 +16,3 @@
 PIBS			= $(shell ls -1d [a-z]*/* | egrep -v 'CVS|Makefile')
 pib_DATA		= $(PIBS)
 SUBDIRS			= $(PIBDIRS)
-
-install-exec-local:
-	for dir in $(PIBDIRS) ; do \
-		$(mkinstalldirs) $(DESTDIR)$(pibdir)/$$dir ; \
-	done
