--- libsmi-0.3.1/lib/smi_config.3.in~	Thu Mar 28 22:25:31 2002
+++ libsmi-0.3.1/lib/smi_config.3.in	Thu Mar 28 22:25:31 2002
@@ -227,7 +227,7 @@
 .fi
 .SH "FILES"
 .nf
-@prefix@/etc/smi.conf    global configuration file
+@sysconfdir@/smi.conf    global configuration file
 $HOME/.smirc               user configuration file
 @prefix@/include/smi.h   SMI library header file
 @mibdir@/     SMI module repository directory
--- libsmi-0.4.1/configure.in~	Thu Mar 28 22:28:05 2002
+++ libsmi-0.4.1/configure.in	Thu Mar 28 22:28:05 2002
@@ -189,7 +189,7 @@
 AM_PROG_LIBTOOL
 AC_SUBST(LIBTOOL_DEPS)
 
-AC_DEFINE_UNQUOTED([DEFAULT_GLOBALCONFIG], "${prefix}/etc/smi.conf",
+AC_DEFINE_UNQUOTED([DEFAULT_GLOBALCONFIG], "${sysconfdir}/smi.conf",
 [The full pathname of the global configuration file.])

 AC_DEFINE([DEFAULT_USERCONFIG], ".smirc",
